vpr/src/place/move_utils.cpp

```diff
@@ -5,6 +5,12 @@
 
 #include "vtr_random.h"
 
+
+//Run-time flag to control when placer debug information is printed
+//Note : only enables debug output if compiled with VTR_ENABLE_DEBUG_LOGGING is defined
+#ifdef VTR_ENABLE_DEBUG_LOGGING
+bool f_placer_debug = true;
+#endif
```

Comment:
This doesn't need to be in the `#ifdef` if we just default it to `false`.

We also probably want to make a command-line option to enable placer debugging (which only does something if compiled with VTR_ENABLE_DEBUG_LOGGING, and gives a warning if specified but not compiled with the option).